access-mgmt/README.adoc

```diff
@@ -0,0 +1,469 @@
+. Click the *Permissions* tab to view the current ACL permissions for this instance.
++
+--
+By default, new Kafka instances contain the following permissions. These permissions allow all accounts in the organization to view the instance permissions and to view topics in the instance, but not to produce or consume messages.
+
+NOTE: The wildcard character `*` denotes any occurrences of the specified resource. For example, `Topic` = `is *` means any topic in a Kafka instance.
+
+.Default ACL permissions for new Kafka instances
+[cols="25%,25%,25%,25%"]
+|===
+h|Account
+h|Resource
+h|Access type
+h|Operation
+
+|`All accounts`
+|`Topic is *`
+
+(Any topic)
+|`Allow`
+|`Describe`
+
+|`All accounts`
+|`Consumer group is *`
+
+(Any consumer group)
+|`Allow`
+|`Describe`
+
+|`All accounts`
+|`Kafka instance`
+
+(Kafka instance permissions in ACLs)
+|`Allow`
+|`Describe`
+|===
+--
+. Use the *Permissions* page to set permissions for a new account, add permissions to an existing account, or delete account permissions in this instance.
++
+To set permissions for a new account in this instance, follow these steps:
+
+.. Click *Manage access*.
+.. In the *Account* drop-down menu, select the new user account or service account that you want to set permissions for. You can also select *All accounts* to set permissions for all user accounts and service accounts in the organization.
+.. Click *Next*.
+.. Under *Review existing permissions*, verify the current permission settings for this instance. You can delete existing permissions now if needed, or you can wait to delete existing permissions later from the main *Permissions* page.
++
+If you previously selected a specific account, you can delete only permission entries that apply to individual accounts. If you previously selected *All accounts*, you can delete only permission entries that apply to all accounts.
+
+.. Under *Assign Permissions*, set the permissions for the specified account or all accounts for a resource type, such as a topic, and click *Add* to continue adding permissions for other resources as needed.
++
+--
+For example, when you create a new service account, you can add the following permissions in the Kafka instance that you want the account to access. In this example, the permissions enable applications associated with the service account to produce and consume messages in any topic in the instance, using any consumer group and any producer.
+
+.Example ACL permissions for a new service account
+[cols="25%,25%,25%,25%"]
+|===
+h|Resource type
+h|Resource identifier
+h|Access type
+h|Operation
+
+|`Topic`
+|`Is` = `*`
+|`Allow`
+|`All`
+
+|`Consumer group`
+|`Is` = `*`
+|`Allow`
+|`Read`
+
+|`Transactional ID`
+|`Is` = `*`
+|`Allow`
+|`All`
+|===
+--
+.. Click *Save* to finish.
+
+To add permissions to an existing account in this instance, follow these steps:
```

Comment:
This gets a bit lost at the end of the current procedure. Worth pulling it out on its own (so, setting permissions for a new instance vs for an existing one)?

Reply:
I went around the block on this: So normally, I'd actually just make the procedure about adding the permissions for a new account, and mention in passing along the way that to hit up an existing one, use the options menu, and to delete, likewise. This is how we've done it with creating a service account, where we say how to do it anew, and have a last step just saying, to update, rename, delete, whatever, go to the Service Accounts page.

However, because managing access tends to be very confusing about what's what, I decided to be a little less concise (again) to make it more laid out. The reason I don't want to go with individual procs is that it's unmerited imo, and unecessarily splits things up into what would then be too narrow of procedures.

So my compromise is not the full streamlined way (all together with brief mentions) and not the full un-streamlined way (individual procs), but a middle-road option of zeroing in on the core task that just has some options. This is also a common approach in the IBM style guide, to make a step of the overall step, with unnumbered listed options and their sub-steps if any.

However, what I should've done and didn't, especially since it's more than two options, is bullet them. I've done that now and it really helps. Wdyt?

![image](https://user-images.githubusercontent.com/29641047/135933985-18806748-6942-48c9-a58c-0840c6b9e179.png)

![image](https://user-images.githubusercontent.com/29641047/135933993-b481b634-8c56-465f-ba5e-d076ec5141cf.png)